### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by noyciy7037**
Add Japanese translation.  
issue:
- properties.png and properties_noclock.png files are not provided. They are not included in the translation files.

### CODE REVIEW

File: RetroBar/Languages/日本語.xaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <s:String x:Key="start_text_xp">スタート</s:String>
+    <s:String x:Key="start_button_tip_98">ここをクリックしてスタート.</s:String>
+    <s:String x:Key="start_button_tip">ここをクリックしてスタート</s:String>
+    <s:String x:Key="start_button_tip_vista">スタート</s:String>
+
+    <s:String x:Key="retrobar_title">RetroBar タスクバー</s:String>
+    <s:String x:Key="toolbars">ツールバー(_T)</s:String>
+    <s:String x:Key="quick_launch">クイック起動(_Q)</s:String>
+    <s:String x:Key="new_toolbar">新しいツールバー(_N)...</s:String>
+    <s:String x:Key="show_taskman_2k">タスクマネージャー(_K)...</s:String>
```
Comment (xoascf) [inline RetroBar/Languages/日本語.xaml:42]: It seems that the name Task Manager had a different spelling (タスク マネージャ) in versions prior to Windows 7:
Are you sure you want to use this modern spelling?

**Comment by xoascf** [comment]
>     properties.png and properties_noclock.png files are not provided. They are not included in the translation files.
Don't worry, I can provide them!

File: RetroBar/Languages/日本語.xaml (hunk 1 above)
Comment (noyciy7037) [reply RetroBar/Languages/日本語.xaml:42]: I checked. "タスク マネージャ" would be more preferable. I will change it.

**Comment by noyciy7037** [comment]
> Don't worry, I can provide them!

Thank you!

**Comment by dremin** [comment]
Thanks!

**Pull request merged**